data/scripts/install-man-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/env sh
+
+# Set PREFIX.
+if [ -z  "${1+x}" ]; then
+    if ! [ -z "${PREFIX+x}" ]; then
+        PREFIX=/usr
+    fi
+else
+    PREFIX=${1}
```

Comment:
`${1}` should be quoted, otherwise the script will fail if the argument includes a space.